### PR TITLE
pre and full resistant distance distributions

### PR DESCRIPTION
**Pull request by Farnazmdi**
@angelaxliu please place the list of full and pre resistant genes in `full = ` and `pre = ` in the cluster function in figure3.

### CODE REVIEW

**Comment by Farnazmdi** [comment]
image

@angelaxliu please add xlabel and ylabel and a title to this figure.

**Review by Farnazmdi — COMMENTED** [review]
I think it is all set.
You can ask Dr. Meyer for review/merge.

**Comment by angelaxliu** [comment]
image

This is the final figure, the mean path lengths are resistant: 0.004462, pre-res: 0.004413, random: 0.004248

File: requirements.txt [hunk 1]
```diff
@@ -16,3 +16,4 @@ scikit-learn==0.24.2
 NetworkX==2.5.1
 pydot==1.4.2
 graphviz==0.16
+bellmanford
```
Comment (aarmey) [inline requirements.txt:19]: You don't need this package as you're using NetworkX, right?

File: de/fitting.py [hunk 2]
```diff
@@ -52,7 +52,7 @@ def runOptim(data, niter=2000, disp=0):
 
     U = np.copy(data)
     np.fill_diagonal(U, 0.0)
-    cost_grad = jit(grad(cost, argnums=0), static_argnums=(3,))
```
Comment (aarmey) [inline de/fitting.py:55]: Is this change necessary?

**Review by aarmey — CHANGES_REQUESTED** [review]
Looks good! Just two minor changes then this should be ready to merge.

File: de/fitting.py (hunk 2 above)
Comment (Farnazmdi) [reply de/fitting.py:55]: I guess so. Since in the last merged PR we no longer have the `linear` argument in the fitting (as it is now handled in another function with Lasso) having the `static_argnums` gives an error when it runs.

Comment (angelaxliu) [reply de/fitting.py:55]: I'm honestly not sure what this does, I don't think it was my edit but I just removed the bellmanford package!

Comment (aarmey) [reply de/fitting.py:55]: Ok, push your edit then I'll approve.

**Review by aarmey — APPROVED** [review]
